app/models/tasks/fix_media_task.rb

```diff
@@ -0,0 +1,33 @@
+class Tasks::FixMediaTask < ::Task
+  before_save :update_media_resource, if: ->(t) { t.status_changed? && t.media_resource }
+
+  def media_resource
+    owner
+  end
+
+  def source_url
+    media_resource&.href
+  end
+
+  def porter_tasks
+    [
+      {
+        Type: "Transcode",
+        Format: "INHERIT",
+        Destination: {
+          Mode: "AWS/S3",
+          BucketName: ENV["FEEDER_STORAGE_BUCKET"],
+          ObjectKey: porter_escape(media_resource.path)
+        },
+        FFmpeg: {
+          OutputFileOptions: "-acodec copy"
```

Comment:
and with a copy, it should be all kinds of fast and not cause any issues with losing quality